toolchains/solidity/core/crates/linter-cli/src/main.rs

```diff
@@ -13,15 +13,7 @@ struct Args {
         default_value = ".",
         help = "Specify project path"
     )]
-    project_path: Vec<String>,
-
-    #[arg(
-        short = 'f',
-        long = "file",
-        default_value = "",
-        help = "Specify a single file to lint"
-    )]
-    file_to_lint: String,
+    project_path: String,
```

Comment:
There should be no project path argument anymore. The file/project path should be retrieve as a positional argument.
Example : 
```
./solidhunter --json path/to/project
```

Reply:
Oh sure, im on it